.github/CONTRIBUTING.md

```diff
@@ -1,23 +1,10 @@
 # Repository contributions
 
 ## Contributions policy
-
-* **Public contributions:** <br/>
-We are currently not accepting public contributions (but we do plan to in 
-later releases and we’ll keep you posted on this). However, we are taking issues and we do want your 
-[feedback](../README.md#give-us-feedback).
-
-* **Improbable developers**: <br/>
-See the [Contributions guide](https://improbableio.atlassian.net/wiki/x/foDrDw).
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your [feedback](../README.md#give-us-feedback).
 
 ## Coding standards
-See the [Unity GDK C# coding standards guide](../docs/contributions/unity-gdk-coding-standards.md).
+See the [GDK for Unity C# coding standards guide](../coding-standards.md).
 
 ## Getting started
-See [About the SpatialOS Unity GDK](../docs/README.md) for
- details on how to get started using the Unity GDK.
-
-## Warning
-This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation -
- see the guidance on
-  [Recommended use](../README.md#recommended-use).
+See [About the SpatialOS GDK for Unity](https://docs.improbable.io/unity/alpha/get-started) for details on how to get started using the Unity GDK.
```

Comment:
reword sentence to 

See [the Get started guide](https://docs.improbable.io/unity/alpha/get-started) for details on how to use the SpatialOS GDK for Unity

Reply:
same for the other one that already got merged in :P